cdap-docs/reference-manual/source/http-restful-api/lifecycle.rst

```diff
@@ -147,7 +147,7 @@ The application's content is the body of the request::
   <JAR binary content>
 
 Invoke the same command to update an application to a newer version.
-However, be sure to stop all of its flows, Spark and MapReduce programs before updating the application.
+However, be sure to stop all of its Spark and MapReduce programs before updating the application.
```

Comment:
Possibly remove "of its" -
be sure to stop all Spark and MapReduce programs...

Reply:
I'll leave "of its" there, since it's not talking about all Spark and MapReduce programs, but just the ones within the application.

Reply:
sg